helmfile.d/values/admin-namespaces-sc.yaml.gotmpl

```diff
@@ -11,6 +11,10 @@ namespaces:
       pod-security.kubernetes.io/warn: restricted
   {{ if .Values.falco.enabled }}
   - name: falco
+    labels:
+      pod-security.kubernetes.io/audit: privileged
+      pod-security.kubernetes.io/enforce: privileged
+      pod-security.kubernetes.io/warn: privileged
```

Comment:
Noticied that we didn't set any psa labels on the falco namespace here which I assume that we want to do.